src/validation/multicityValidation.js

```diff
@@ -0,0 +1,33 @@
+import { body, validationResult } from 'express-validator';
+
+const multicityCheck = [
+  body('departureDate').trim().not().isEmpty()
+    .withMessage('Departure Date field is required')
+    .matches(/^\d{4}([-./,:])\d{2}\1\d{2}$/, 'i')
+    .withMessage('The date must follow date format YYYY-MM-DD'),
+  body('returnDate').trim().not().isEmpty()
+    .withMessage('Return Date field is required')
+    .matches(/^\d{4}([-./,:])\d{2}\1\d{2}$/, 'i')
+    .withMessage('The date must follow date format YYYY-MM-DD'),
+  body('currentOfficeLocation').trim().not().isEmpty()
+    .withMessage('current Office Location field is required'),
+  body('reason').trim().not().isEmpty()
+    .withMessage('Reason field is required'),
+  body('tripType').trim().not().isEmpty()
+    .withMessage('The trip type field is required')
+    .matches(/^Multi-city$/, 'i')
```

Comment:
@sebaztyn this should be `multicity` and not `Multi-city`

Reply:
I have resolved this with @Stylll